content/en/docs/zero-code/java/spring-boot-starter/_index.md

```diff
@@ -23,3 +25,34 @@ OpenTelemetry.
    - **Spring Boot configuration files** (`application.properties`,
      `application.yml`) to configure the OpenTelemetry Spring Boot starter which
      doesn't work with the OpenTelemetry Java agent
+
+## Comparison to Micrometer
+
+At a high level, this section compares OpenTelemetry Spring Boot starter and
+Micrometer.
+
+OpenTelemetry Spring Boot starter:
+
+- Natively uses the OpenTelemetry API, OpenTelemetry SDK, and the OpenTelemetry
+  semantic conventions.
+- Supported by the OpenTelemetry community (governed by the CNCF / Linux
+  Foundation).
+
+Micrometer [^1][^2]:
+
+- An independent standalone observability system.
+- Offers optional bridges into parts of the OpenTelemetry ecosystem.
+- Supported by the Micrometer community (governed by Broadcom Inc).
+
+As the authors of the OpenTelemetry Spring Boot starter, we recommend using the
+OpenTelemetry Spring Boot starter. We have a great community behind it, and are
+continuously making improvements. If you try it out and have any problems or
+suggestions, open an
+[issue](https://github.com/open-telemetry/opentelemetry-java-instrumentation/issues)
+in our repository.
```

Comment:
```suggestion
Both solutions are open source and support different exporters, including OTLP and Prometheus,
and both are compatible for tracing use cases. This means, that they fit equally into an existing 
observability architecture. It is up to the end user to decide, which solution satisfies their requirements better. 
```

This is the sentence I would remove in the light of "neutrality". It's very obvious that we recommend the otel solution, so we don't have to say that explicitly.

Reply:
It sounds a bit like we would not be convinced that our solution is the better one.

We do try to become the standard API in the java world - can we say that in a friendly way?